modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -371,6 +371,32 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
 };
 /** @} */
 
+namespace gapi {
+namespace streaming {
+/**
+ * @brief Ask G-API to set specific queue capacity for streaming execution.
+ *
+ * For streaming execution G-API has a special concurrent bounded queue to
```

Comment:
For -> In the

G-API has -> , G-API maintains (note the comma)

queue -> queues.

Reply:
Done